vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/index/CaseSensitiveAnalyzer.java

```diff
@@ -38,7 +38,7 @@
  * Filters {@link StandardTokenizer} with {@link StandardFilter},
  * and {@link StopFilter}, using a list of
  * English stop words.
- * <a name="version"/>
+ * <a> name="version"</a>
```

Comment:
I am not sure why this was not allowed, but this is not a correct fix.
Try this: `<a name="version"></a>`